### ISSUE TITLE
Suggestion: Month/Week tracking

### ISSUE THREAD

**Issue by Grygon** [luetkemj/aglet-timekeeper#4]
It'd be cool if this could track months/weeks, particularly if it also had the option to set custom month/week lengths. Similarly, a customization option for the lunar cycle would be awesome for those who have homebrewed worlds,

**Comment by luetkemj** [comment]
Yeah, this would be super cool and has been the most requested feature. That said it's a rather high effort and will take some thought before I can begin implementing it.
